sdcm/nemesis.py

```diff
@@ -1831,12 +1831,21 @@ def repair_nodetool_rebuild(self):
 
     def disrupt_nodetool_cleanup(self):
         # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
-        test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+        # test_keyspaces = self.cluster.get_test_keyspaces()
+
+        # Inner disrupt function for ParallelObject
+        def _dis_nodetool_cleanup(node):
             InfoEvent('NodetoolCleanupMonkey %s' % node).publish()
             with adaptive_timeout(Operations.CLEANUP, node, timeout=HOUR_IN_SEC * 48):
-                for keyspace in test_keyspaces:
-                    node.run_nodetool(sub_cmd="cleanup", args=keyspace)
+                # for keyspace in test_keyspaces:
+                node.run_nodetool(sub_cmd="cleanup")
+
+        threads = ParallelObject(self.cluster.nodes, num_workers=min(32, len(self.cluster.nodes)))
+        thread_results = threads.run(_dis_nodetool_cleanup, ignore_exceptions=True, unpack_objects=True)
```

Comment:
Why are we ignoring exceptions ?

And then not failing the nemesis when finding errors ?

We shouldn't be ignoring failures, we should let them bubble up and fail the nemesis.

Reply:
OK, then I change it to default value ignore_exceptions=False

Reply:
However, I'm not sure if the thread_results needed and what to check in it as if anything goes wrong exception will be raised anyway.